lib/appmap/util.rb

```diff
@@ -121,15 +121,16 @@ def select_rack_headers(env)
           blank?(headers) ? nil : headers
         end
 
-        # Rack prepends HTTP_ to all client-sent headers.
+        # Rack prepends HTTP_ to all client-sent headers (except Content-Type and Content-Length?).
```

Comment:
It seemed so bizarre to me that these two headers would be treated differently, I went off and researched why.

Turns out rack is just following the CGI spec: https://datatracker.ietf.org/doc/html/rfc3875#section-4.1.18 .

Will you add a comment to that effect, please, and include that link to the RFC?